packages/espresso/example/README.md

```diff
@@ -8,7 +8,7 @@ The espresso package only runs tests on Android. The example runs on iOS, but th
 
 To run the Espresso tests:
 
-```
+```java
```

Comment:
This was the only violation of a pre-existing README check in an example README.